apps/system/index.html

```diff
@@ -413,6 +409,14 @@
     <script defer src="js/suspending_app_priority_manager.js"></script>
     <script defer src="js/input_window.js"></script>
 
+    <!-- App Install -->
+    <link rel="stylesheet" type="text/css" href="style/app_install_manager/app_install_manager.css">
+    <script defer src="js/app_install_dialog.js"></script>
+    <script defer src="js/app_install_cancel_dialog.js"></script>
+    <script defer src="js/app_download_cancel_dialog.js"></script>
+    <script defer src="js/setup_installed_app_dialog.js"></script>
+    <script defer src="js/ime_layout_dialog.js"></script>
```

Comment:
All of these should be lazy loaded by AppInstallManager instead of putting here.